alpaca/rest_test.go

```diff
@@ -602,6 +602,40 @@ func TestGetAccountActivities(t *testing.T) {
 	_, err = c.GetAccountActivities(&dividendsActivityType, nil)
 	assert.NotNil(t, err)
 	assert.EqualError(t, &APIError{Code: 500, Message: "internal server error"}, "internal server error")
+
+	// test filter by date and URI
+	c.do = func(c *client, req *http.Request) (*http.Response, error) {
+		getQuery := req.URL.Query()
+
+		assert.Equal(t, "/v2/account/activities/DIV", req.URL.Path)
+		assert.Equal(t, "2019-01-01T00:00:00-07:00", getQuery.Get("after"))
```

Comment:
As we discussed previously, the timezone depends on your actual location. This is why this unit test fails on the CI. 

My suggestion: 
``` go
opts.Date.UTC().Format(time.RFC3339)
```
Use UTC regardless of the timezone provided by the client.